### ISSUE TITLE
"Suble knockout" [name pending]

### ISSUE THREAD

**Issue by Paulorpribeiro** [mlenser/kryx-rpg-issues#200]
Maneuver to be modeled similar to the sleep spell, which I quote below:

"As an action, you send creatures into a magical slumber. Roll 5d10; the total is how many hit points of creatures this spell can affect. Creatures within a sphere centered on a point you choose within 120 feet are affected in ascending order of their current hit points (ignoring unconscious creatures)."

Only it should be single target, not magical slumber but just unconscious, and probably not 5d10 hp, but something appropriate to maneuver "damage budget" from the balance spreadsheet.
I would also envision this maneuver to be low level (maybe one or two stamina) with augments to simply increase the dice rolled for hp. I also like the part of the spell that says the creature is unconscious until it takes damage or someone slaps it away.
This would be a deceit maneuver

**Comment by mlenser** [comment]
I don't believe a maneuver can fit the same model. Sleep, and similar spells, target multiple creatures and there is an expected amount of failure involved as creatures might have different HP than expected.

Unconscious is far more powerful than incapacitated. A maneuver that uses it would be ~5 stamina dice.

Sleep spells were designed in a way to give very powerful conditions at a lower cost. I don't think the same is really possible for a maneuver.

**Comment by Paulorpribeiro** [comment]
If I remember correctly it was a 1 or 2 stamina maneuver back then. I was thinking along the lines of "roll twice the stamina dice you spend in this maneuver. If the remaining hp of the target is lower, it is unconscious until dealt damage or slap awake." If this is too strong, perhaps make prerequisite that the target is unaware of the attacker. This is supposed to be surprise attack.

**Comment by itamarcu** [comment]
Sounds like you're basically describing a maneuver for "deal more damage, but it is nonlethal, and if it doesn't make the target unconscious it doesn't do anything".

I think this could work, though it would be quite disappointing for a player that uses this on an enemy with more hp than required, as it would be a complete waste.

Perhaps it could be an augment to one of the stunning/staggering maneuvers, that says the target is knocked unconscious if its remaining hit points are less than some amount?

**Comment by mlenser** [comment]
> Perhaps it could be an augment to one of the stunning/staggering maneuvers, that says the target is knocked unconscious if its remaining hit points are less than some amount?

Didn't you just invalidate this option with what you wrote immediately above it?

> it would be quite disappointing for a player that uses this on an enemy with more hp than required, as it would be a complete waste.

The spells are quite different as they are very limited:
- Target a specific amount of HP
- Always start with the lowest hp
- Extra HP is lost
- Can be broken easily

A maneuver can't achieve several of those. It can only achieve:
- Extra HP is lost
- Can be broken easily

Based on that and the issue of it not applying if the creature has more hp than expected I don't think we can create maneuvers that bring powerful conditions to earlier tiers by using HP as a gate.

**Comment by Paulorpribeiro** [comment]
Well, I agree with you. Perhaps the previous maneuver was overpowered, maybe that is why it was chopped. But I still think the deceive or shadow theme should have a knockout maneuver. Maybe we can copy the current knockout from earth/vanguard and chande the flavour?
By the way, I think the current knockout maneuver should be restricted to bludgeoning weapons/attacks

**Comment by Paulorpribeiro** [comment]
Well, I agree with you. Perhaps the previous maneuver was overpowered, maybe that is why it was chopped. But I still think the deceive or shadow theme should have a knockout maneuver. Maybe we can copy the current knockout from earth/vanguard and chande the flavour?
By the way, I think the current knockout maneuver should be restricted to bludgeoning weapons/attacks

**Comment by mlenser** [comment]
Knockout maneuver only works on bludgeoning damage based attacks now
Knockout maneuver added to the Deceit theme
